### ISSUE TITLE
Accesibility Audit

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/gutenberg-blocks#109]
I want to do an accessibility audit of Otter and find the issues with it and then fix them. It's better to think of accessibility of the plugin from the beginning so we don't have to "hack" things into it later.

If we do it now:

- The code will be better
- Will be really helpful to people suffering disabilities
- A good way to promote
- I will also learn about a11y which will help us in the future
- It will be harder in future and easier when the plugin is new

So far the plugin is accessible in terms of keyboard navigation but not in terms of accessible HTML code.

The idea is to first learn accessibility guidelines (maybe a course or reading articles, or maybe a book or docs) and then implement. I suspect it to take around two weeks of time (could be way more or less but that's my current estimate).

And once we are done we can maybe hire someone from Upwork to do a small audit?

I personally find accesibility to be a priority and would like to get your perspectives. @selul

**Comment by selul** [comment]
@HardeepAsrani yes, sounds good to me.  Do we have any new released planned for Otter? 
If so we can tackle this afterwards, otherwise, we can start looking into this.

We can find someone to audit the plugin current state and see what we can improve. 
After we see what shape we are we can decide how much time we can invest in this. 

As a side note, we should probably look for someone that is pretty aware of the WordPress ecosystem too for the audit.

**Comment by HardeepAsrani** [comment]
@selul We don't have any major release planned for now. We have a minor release that will go out today. How do you think we should reach out to a11y reviewers? Also, do we have a budget?

**Comment by HardeepAsrani** [comment]
I've sent a tweet to see if we find someone from Twitter. Will connect with you if someone connects.
